README.md

```diff
@@ -176,7 +178,7 @@ jwt.sign({ userId: user.id })
 
 ## Security
 
-This microservice is intended to be very secure.
+This microservice is intented to be very secure.  User accounts can be limited to certain domains by configuring the `WHITELISTED_DOMAINS` env variable.
```

Comment:
`intented` -> `intended`

I actually fixed this one earlier today, but I think git just automerged the conflict.